### ISSUE TITLE
Is there any way to add support for ios9?

### ISSUE THREAD

**Issue by hendrysadrak** [pixijs/pixi-sound#19]
Currently getting error `Error: InvalidStateError: DOM Exception 11` when calling `PIXI.sound.play('soundName');`

**Comment by bigtimebuddy** [comment]
Is this error thrown with the [examples](https://pixijs.github.io/pixi-sound/examples/)?

**Comment by hendrysadrak** [comment]
The examples dont work because of ES6 syntax :D

**Comment by bigtimebuddy** [comment]
Looks like a pretty easy fix. I added this to the **dev** branch, but will also probably make a patch release to master for this. The offending line is here:

https://github.com/pixijs/pixi-sound/blob/master/src/SoundInstance.ts#L245

```js
this._source.start(0, start, (end ? end - start : undefined));
```

For iOS 9, this should be:

```js
if (end) {
  this._source.start(0, start,  end - start);
} else {
  this._source.start(0, start);
}
```

**Comment by hendrysadrak** [comment]
Perfect! Thank you for the quick response! When will you merge dev to master or create a patch?

**Comment by bigtimebuddy** [comment]
Released in [v1.4.2](https://github.com/pixijs/pixi-sound/releases/tag/v1.4.2)
